src/ocamlorg_toplevel/bin/dune

```
@@ -0,0 +1,73 @@
+; Base
+
+(executable
+ (name toplevel_base)
+ (libraries ocamlorg_toplevel base)
+ (modules toplevel_base)
+ (flags
+  (:standard -rectypes -linkall))
+ (modes byte))
+
+(rule
+ (targets export_base.txt)
+ (deps toplevel_base.bc)
+ (action
+  (run jsoo_listunits -o %{targets} stdlib base)))
+
+(rule
+ (targets toplevel_base.js)
+ (action
+  (run
+   %{bin:js_of_ocaml}
+   --export
+   %{dep:export_base.txt}
+   --toplevel
+   --pretty
+   +toplevel.js
+   +dynlink.js
+   +base/runtime.js
+   %{dep:toplevel_base.bc}
+   -o
+   %{targets})))
+
+; Stdlib
+
+(executable
+ (name toplevel_stdlib)
+ (libraries ocamlorg_toplevel stdlib)
+ (modules toplevel_stdlib)
+ (flags
+  (:standard -rectypes -linkall))
+ (modes byte))
+
+(rule
+ (targets export_stdlib.txt)
+ (deps toplevel_stdlib.bc)
+ (action
+  (run jsoo_listunits -o %{targets} stdlib)))
+
+(rule
+ (targets toplevel_stdlib.js)
+ (action
+  (run
+   %{bin:js_of_ocaml}
+   --export
+   %{dep:export_stdlib.txt}
+   --toplevel
+   --pretty
+   +toplevel.js
+   +dynlink.js
+   %{dep:toplevel_stdlib.bc}
+   -o
+   %{targets})))
+
+(subdir
+ js/
+ (rule
+  (alias toplevel)
+  (targets stdlib-4.12.0.js base-v0.14.1.js)
+  (deps ../toplevel_stdlib.js ../toplevel_base.js)
+  (action
+   (progn
+    (copy ../toplevel_stdlib.js stdlib-4.12.0.js)
+    (copy ../toplevel_base.js base-v0.14.1.js)))))
```

Perhaps there's some way to make the jsoo compiler do this automatically? At any rate currently this seems to half the size of the files (uncompressed). 

```suggestion
    (run jsoo_minify ../toplevel_stdlib.js -o stdlib-4.12.0.js)
    (run jsoo_minify ../toplevel_base.js -o base-v0.14.1.js)))))
```